streamz/dataframe/core.py

```diff
@@ -270,17 +271,19 @@ class DataFrame(Frame, _DataFrameMixin):
     """
 
     def __init__(self, *args, **kwargs):
-        # {'x': sdf.x + 1, 'y': sdf.y - 1} - works only with pandas
+        # {'x': sdf.x + 1, 'y': sdf.y - 1}
         if len(args) == 1 and not kwargs and isinstance(args[0], dict):
-            def concat(tup, columns=None):
-                result = pd.concat(tup, axis=1)
+            def concat(tup, module=None, columns=None):
```

Comment:
Instead of making a function inline, would it not make sense to store the module as an instance attribute, and have this as a method? Then you don't have to call get_dataframe_package multiple times.

Reply:
Right, OK - that's annoying!